_includes/configuration/available-context-menu-items.md

```diff
@@ -0,0 +1,22 @@
+## Available context menu items
+
+The following table shows all context menu items, including items provided by plugins.
+Any menu item can also be added to the context menu.
```

Comment:
This paragraph doesn't read very clearly to me tbh. Is this actually referring to context menu *items* or is it *sections* or a mix of both? Maybe... there needs to be a sentence describing the difference between a normal menu item and what we're referring to here as context menu items?

Reply:
@metricjs I've made some changes on https://github.com/tinymce/tinymce-docs/pull/1495/commits/b4c066d3878670242676aecd57a8621fdd9e269f let me know what you think

Reply:
Looks good!